src/semant/translate.sml

```diff
@@ -150,4 +151,7 @@ structure Translate : TRANSLATE = struct
                                  T.LABEL(join)]),
                                T.CONST(0))))
     end
+
+  fun sequence(l: exp list) =
+    Ex(Tree.ESEQ(l))
```

Comment:
Yes it does... In that case, this should probably just be a recursive function which individually adds each ```exp```